locales/it/index.yml

```
@@ -1858,6 +1858,7 @@ bonus:
           title: "Attivazione metodo"
           body: "Confermi di voler attivare il cashback su questo metodo di pagamento?
                  L’attivazione sarà operativa **da domani**, quindi non saranno valide ai fini del cashback le transazioni che farai nella giornata di oggi."
+          disclaimer: "Cliccando su Attiva, dichiari di essere il titolare di questo metodo di pagamento."
```

```suggestion
          disclaimer: "Cliccando su {{ activate }}, dichiari di essere il titolare di questo metodo di pagamento."
```

```typescript
{I18n.t(
  "bonus.bpd.details.paymentMethods.activate.disclaimer",
  { activate: I18n.t("global.buttons.activate") }
)}
```

Could be an improvement to avoid mismatching if the CTA text changes?

---

changed in 3bc194f